### DISCUSSION TITLE
Не могу добавить  папки в репозиторий, в чем может быть дело?

### DISCUSSION THREAD

**Discussion by EgorAlymov** [community/community#143958] [Repositories]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

Не могу добавить  папки в репозиторий, в чем может быть дело?

**Comment by queenofcorgis** [comment]
Hi @EgorAlymov , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be closing and locking any posts in languages other than English for now, including this one.

Please note for future posts: You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.
